scripts/src/view/input/fugaziInput.tsx

```diff
@@ -116,21 +117,37 @@ export class FugaziInputView extends base.SuggestibleInputView<FugaziInputProper
 	}
 
 	protected onSuggestionItemPressed(item: statements.Statement): void {
-		let newValue: string = "",
+		let oldValueWords: string[] = this.getValue().split(" "),
+			expressionsIterator = (item.getExpression() as input.CommandExpression).getExpressions().getIterator(),
+			newValue: string = "",
 			haveStoppedForParameter = false,
 			iterator = item.getRule().getTokens().getIterator();
+
 		while (iterator.hasNext()) {
 			let token: syntax.RuleToken = iterator.next();
```

Comment:
refactor: you can probably remove the type annotation:
```
let token = iterator.next();
```
also, looks like it can be a `const`, no?